pkg/replication/replication.go

```diff
@@ -416,7 +416,10 @@ func (r *replication) updateOne(
 	timer := time.NewTimer(exponentialBackoff)
 	for err != nil {
 		nodeLogger.WithError(err).Errorln("Could not write intent store")
-		r.errCh <- err
+		select {
+		case r.errCh <- err:
```

Comment:
any reason not to move this into the lower select? that way we only drop the error if this has to take another action